alpha/apps/kaltura/lib/myCloudUtils.class.php

```diff
@@ -22,4 +23,11 @@ public static function getSharedTempBucket()
 	{
 		return kConf::get(self::SHARED_TEMP_BUCKET, self::CLOUD_STORAGE_MAP, null);
 	}
+
+	public static function shouldExportThumbToCloud()
+	{
+		$exportRatio = kConf::get(self::THUMB_EXPORT_RATIO_PARAM, self::CLOUD_STORAGE_MAP, 0);
+		$random = rand(0, 99);
```

Comment:
Better use mt_rand, in version older than 7.1 , it might produce the same number again and again....